patches/tModLoader/Terraria/ModLoader/Core/ModOrganizer.cs

```diff
@@ -44,40 +45,42 @@ internal static class ModOrganizer
 			modRepos.AddRange(WorkshopFileFinder.ModPaths);
 
 			foreach (string repo in modRepos) {
-				foreach (string fileName in Directory.GetFiles(repo, "*.tmod", SearchOption.TopDirectoryOnly)) {
-					var lastModified = File.GetLastWriteTime(fileName);
-
-					if (!modsDirCache.TryGetValue(fileName, out var mod) || mod.lastModified != lastModified) {
-						try {
-							var modFile = new TmodFile(fileName);
-
-							using (modFile.Open()) {
-								mod = new LocalMod(modFile) {
-									lastModified = lastModified
-								};
-							}
+				var fileName = GetActiveTmodInRepo(repo);
```

Comment:
I think this breaks loading multiple (or any) mods from the `Mods` (dev builds) repo